components/fhir-common/src/main/java/org/hl7/gravity/refimpl/sdohexchange/fhir/parse/BundleParser.java

```diff
@@ -0,0 +1,24 @@
+package org.hl7.gravity.refimpl.sdohexchange.fhir.parse;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Resource;
+
+public abstract class BundleParser<T> {
```

Comment:
This looks more like extractor or something) parser is for parsing, e.g. working with Strings

Reply:
Renamed.